src/views/CreateShareStructure.vue

```diff
@@ -1,18 +1,346 @@
 <template>
-   <section class="mt-10">
-      <header id="share-structure-header">
-        <h2>1. Create Your Authorized Share Structure</h2>
-      </header>
+  <section class="mt-10">
+    <header id="share-structure-header">
+      <h2>1. Create Your Authorized Share Structure</h2>
+    </header>
+    <p>
+      Add at least one class of shares. A share class consists of the name of the class, the maximum number of shares in
+      the class (including any associated Series), a par value for the class, and the currency the shares are valued in.
+    </p>
+
+    <!-- Help Section -->
+    <span class="help-btn" @click="helpToggle = !helpToggle">
+      <v-icon color="#1976d2" style="padding-right: 5px">mdi-help-circle-outline</v-icon>
+      <span v-if="!helpToggle">Help with Share Structure</span>
+      <span v-else>Hide Help</span>
+    </span>
+    <section v-show="helpToggle" class="share-structure-help">
+      <header id="share-structure-help-header"><h2>Share Structure Help</h2></header>
+      <p>
+        An incorporated business must issue shares. These shares represent ownership interest in the company and give
+        the shareholder a say in how the company is being run. For most small companies starting out, a simple share
+        structure with just one class of shares (and no series) is typical.
+      </p>
+      <p>
+        The sample below is for a typical share structure with just one class of shares. If there is more than one class
+        of shares, each class must be assigned an identifying name such as Class A, Class B, etc.
+      </p>
+      <div id="share-structure-sample">
+        <ListShareClass :shareClasses="sharesHelpSample"/>
+      </div>
+      <p><small>Sample Share Structure</small></p>
+      <h4>Important Information About the Company's Share Structure</h4>
```

Comment:
Sorry for being picky, but this H4 follows an H2 (on line 4). This goes against accessibility guidelines (where accessibility tools try to lay out a page's structure using Header tags, but this confuses them). This should be a H3. If you need to style it smaller, so be it.

Reply:
I tried h3. It looks very big and odd. h4 looks better here

Reply:
Can you change this to a H3 and style it as an H4? (There may be a class name "h4".)

Reply:
Will leave it as such for now